### PR TITLE
Beautify console output - Closes #26

### LINKED ISSUE #26: Beautify console output
Advancing the help pages, adding logo and pictures into the initial opening etc.

### PR DESCRIPTION
**Pull request by Tosch110**
Closes #26

### CODE REVIEW

File: src/utils/tablify.js [hunk 1]
```diff
@@ -44,7 +62,14 @@ const tablify = (data) => {
 		? data.reduce(reduceKeys, [])
 		: getKeys(data);
 
-	const table = new Table({ head });
+	const table = new Table({
+		head,
+		chars,
+		style: {
+			head: ['cyan'], // disable colors in header cells
```
Comment (willclarktech) [inline src/utils/tablify.js]: Comments

File: test/specs/commands/createTransactionCreateMultisignatureAccount.js [hunk 2]
```diff
@@ -29,12 +29,12 @@ describe('create transaction create multisignature account command', () => {
 							Given('an empty options object', given.anEmptyOptionsObject, () => {
 								Given('a lifetime of "NaN" hours', given.aLifetimeOfHours, () => {
 									When('the action is called with the keysgroup, the lifetime, the minimum number of signatures and the options', when.theActionIsCalledWithTheKeysgroupTheLifetimeTheMinimumNumberOfSignaturesAndTheOptions, () => {
-										Then('it should reject with message "Lifetime must be an integer."', then.itShouldRejectWithMessage);
+										Then('it should reject with message "Lifetime must be an integer."', then.itShouldRejectWithValidationErrorAndMessage);
```
Comment (willclarktech) [inline test/specs/commands/createTransactionCreateMultisignatureAccount.js]: Descriptions should be updated

File: test/specs/utils/error.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import * as given from '../../steps/1_given';
+import * as when from '../../steps/2_when';
+import * as then from '../../steps/3_then';
+
+describe('Custom Error Message', () => {
+	Given('a function that throws a validation error "could not be validated"', given.aFunctionThatThrowsAValidationError, () => {
+		When('the validation error is thrown', when.validationErrorIsThrown, () => {
```
Comment (willclarktech) [inline test/specs/utils/error.js]: Description and function name diverge

File: test/specs/utils/error.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+			Then('it should print the error message in red', then.itShouldPrintTheErrorMessageInRed);
+			Then('the error should have the name "ValidationError"', then.theErrorShouldHaveTheName);
+			Then('the error should be instance of build in error', then.theErrorShouldBeInstanceOfBuildInError);
```
Comment (willclarktech) [inline test/specs/utils/error.js]: `should be an instance of Node’s built-in Error`

File: test/specs/utils/helpers.js [hunk 5]
```diff
@@ -27,12 +27,12 @@ describe('utils helpers', () => {
 		});
 		Given('an invalid string lifetime of "123.4" hours', given.anInvalidStringLifetimeOfHours, () => {
 			When('validateLifetime is called on the lifetime', when.validateLifetimeIsCalledOnTheLifetime, () => {
-				Then('it should throw error "Lifetime must be an integer."', then.itShouldThrowError);
+				Then('it should throw error "Lifetime must be an integer."', then.itShouldThrowValidationError);
```
Comment (willclarktech) [inline test/specs/utils/helpers.js]: Descriptions should be updated

File: src/utils/error.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+export const configurationWarningMessage = (path) => {
+	const warning = `WARNING: Could not write to \`${path}\`. Your configuration will not be persisted.`;
+	console.warn(chalk.yellow(warning));
+	return null;
+};
+
+export const fileSystemErrorMessage = (errorMessage) => {
+	console.error(chalk.red(errorMessage));
```
[the remark this answers is not in the document]
Comment (Tosch110) [reply src/utils/error.js]: I actually prefer not to override existing methods and build our own instead.

[the remark this answers is not in the document]
Comment (willclarktech) [reply src/utils/error.js]: OK, can we have a function that still allows string substitution though?

**Pull request merged**